app/controllers/reservations/tickets_controller.rb

```diff
@@ -0,0 +1,19 @@
+module Reservations
+  class TicketsController < ApplicationController
+    def index
+      @tickets = Tickets::UseCases::FetchAll.new(reservation_id: params[:reservation_id]).call
```

Comment:
```suggestion
      tickets = Tickets::UseCases::FetchAll.new(reservation_id: params[:reservation_id]).call
```

Why do you use the **instance variable** here? Do you have a problem with understanding the concept of instance and local variables?
http://ruby-for-beginners.rubymonstas.org/writing_classes/instance_variables.html

Reply:
🙈 You are right... I had some troubles but not with understanding what it is. I am used to writing in Railsway where I have to use instance variables so that I can use them in HTML. Anyway - fixed! 🎉